pkg/ndn-riot/patches/0003-adapt-to-moved-kernel_pid_t-location.patch

```diff
@@ -0,0 +1,39 @@
+From 51d973f2a0a04a716af8260c62fb5045356a30a5 Mon Sep 17 00:00:00 2001
+From: Kaspar Schleiser <kaspar@schleiser.de>
+Date: Mon, 23 Nov 2020 12:44:54 +0100
+Subject: [PATCH] adapt to moved kernel_pid_t location
+
+---
+ face-table.h | 2 +-
+ ndn.h        | 2 +-
+ 2 files changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/face-table.h b/face-table.h
```

Comment:
Can you do a PR to https://github.com/named-data-iot/ndn-riot as a follow-up?
It feels a bit silly to carry downstream patches to a project that targets RIOT exclusively.

Reply:
yup!

https://github.com/named-data-iot/ndn-riot/pull/14

Reply:
We should probably do the same for the other two patches in `pkg/ndn-riot/patches`, those also stem from previous API changes in RIOT